TypeInjections/Tool.Test/Resources/Git/BadLocation/file2.dfy

```diff
@@ -0,0 +1,13 @@
+
+module S {
+
+
+
+// BOOM NOW YOU SEE ME
```

Comment:
unnecessary whitespace above and the comment doesn't help

Reply:
This is apart of the remote repository test that we want to remove in favor of mockup. I will delete all these files